### PR TITLE
Update icons to current Structure 1.9

### PR DESCRIPTION
**Pull request by nfagerlund**
In Figma: https://www.figma.com/file/B7yd8loYS7nA2TiQcJ8UtV/Structure-1.9?node-id=0%3A64

It looks like Structure 1.9 may have been updated in-place over time, and now contains a few significant changes from the published 1.9.0-0 version of this package. In particular, there's a revised version of an "unlocked" icon that @roarytubbs wants me to use in a current WIP. 

~This PR also deletes some icons that are no longer included in the canonical Figma document for Structure.~ (Never mind, let's not do that.)

~It does NOT update the logos at all, because they weren't in the .zip I got from Roary and I apparently don't have export permissions in Figma.~ (I got updated logo SVGs!)

### CODE REVIEW

**Comment by Dhaulagiri** [comment]
> This PR also deletes some icons that are no longer included in the canonical Figma document for Structure.

Would this be a breaking change for apps that were built on designs that did include those icons?

**Comment by roarytubbs** [comment]
> > This PR also deletes some icons that are no longer included in the canonical Figma document for Structure.
> 
> Would this be a breaking change for apps that were built on designs that did include those icons?

I would agree here. Is there a reason we should delete these icons?

**Comment by nfagerlund** [comment]
@Dhaulagiri @roarytubbs Mmm, I suppose not? I was assuming the Figma document was the canonical record of which icons SHOULD be used, but on further thought, it probably doesn't matter if some older icons stick around.

File: src/Logo - BitBucket - Color.svg [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
 <path fill-rule="evenodd" clip-rule="evenodd" d="M2.17488 3.24048C2.29487 3.0999 2.47074 3.01914 2.65575 3.01963L21.0492 3.02291C21.2342 3.02241 21.4101 3.10318 21.5301 3.24375C21.6501 3.38432 21.702 3.57047 21.6721 3.75273L19.0164 20.1425C18.9667 20.4476 18.7031 20.6719 18.3935 20.6727H5.53444C5.1241 20.6669 4.77523 20.3721 4.70165 19.9691L2.0328 3.74945C2.00293 3.5672 2.05489 3.38105 2.17488 3.24048ZM9.80657 14.7425H13.9213L14.9148 8.95636H8.71149L9.80657 14.7425Z" fill="#2684FF"/>
-<path d="M20.8328 8.95636H14.9312L13.9476 14.7425H9.80659L4.98364 20.4764C5.13707 20.6079 5.33222 20.681 5.53446 20.6825H18.3935C18.7031 20.6818 18.9667 20.4574 19.0164 20.1524L20.8328 8.95636Z" fill="url(#paint0_linear)"/>
+<path d="M20.8328 8.95636H14.9311L13.9475 14.7425H9.80653L4.98358 20.4764C5.13701 20.6079 5.33215 20.681 5.5344 20.6825H18.3934C18.7031 20.6818 18.9666 20.4574 19.0164 20.1524L20.8328 8.95636Z" fill="url(#paint0_linear)"/>
 <defs>
-<linearGradient id="paint0_linear" x1="17.0512" y1="6.95832" x2="10.2342" y2="16.6596" gradientUnits="userSpaceOnUse">
+<linearGradient id="paint0_linear" x1="17.0511" y1="6.95832" x2="10.2342" y2="16.6596" gradientUnits="userSpaceOnUse">
```
Comment (meirish) [inline src/Logo - BitBucket - Color.svg:5]: Seems like this would trigger the duplicate `id` issue we'd seen previously.

**Review by meirish — APPROVED** [review]
I ran the preview and all of these look good to me - @roarytubbs are you good with these too? If so we can merge and I'll do a release.

**Comment by roarytubbs** [comment]
> I ran the preview and all of these look good to me - @roarytubbs are you good with these too? If so we can merge and I'll do a release.

As long as we aren't deleting icons I'm good with it.  @meirish

**Pull request merged**